Source/JavaScriptCore/wasm/WasmOperations.cpp

```diff
@@ -303,14 +306,21 @@ JSC_DEFINE_JIT_OPERATION(operationWasmToJSExitMarshalArguments, bool, (void* sp,
     VM& vm = instance->vm();
     auto scope = DECLARE_THROW_SCOPE(vm);
 
-    constexpr int codeBlockOffset = -0x18;
-
-    auto* importableFunction = *access.operator()<WasmOrJSImportableFunctionCallLinkInfo*>(cfr, codeBlockOffset);
+    auto* importableFunction = *access.operator()<WasmOrJSImportableFunctionCallLinkInfo*>(cfr, WasmCallableFunctionScratch);
     auto typeIndex = importableFunction->typeIndex;
     const TypeDefinition& typeDefinition = TypeInformation::get(typeIndex).expand();
     const auto& signature = *typeDefinition.as<FunctionSignature>();
     unsigned argCount = signature.argumentCount();
 
+    // Set up bookkeeping in case we throw an exception here.
```

Comment:
I am not sure why this was working before; If we threw an exception below, we wouldn't have a valid callee